### ISSUE TITLE
Screen Reader: Workspace: The button of more options has no button role announced

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76910]
## Action Performed:
Prerequisites:
- the user is signed in

1. Using NVDA+Firefox, open the URL https://new.expensify.com/
2. Navigate using the Tab key to the Workspaces section. and press the enter key to activate it.
3. Tab to focus the More options icon button (three vertical dots).
4. Observe the screen reader announcement.

## Expected Result:
The control should be announced with a button role.

## Actual Result:
The screen reader does not announce any actionable role when the control More options icon (three vertical dots). When focused, the screen reader announces only the label of the control as “More options”. Hence the screen reader users will not be aware that the control is actionable and can be activated.

HTML:
<div aria-label="More" role="presentation" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-1awozwy r-eu3ka r-1777fci r-1aockid" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#A2A9A3"><path d="M12.5 3c0 1.4-1.1 2.5-2.5 2.5S7.5 4.4 7.5 3 8.6.5 10 .5s2.5 1.1 2.5 2.5m0 7c0 1.4-1.1 2.5-2.5 2.5S7.5 11.4 7.5 10 8.6 7.5 10 7.5s2.5 1.1 2.5 2.5M10 19.5c1.4 0 2.5-1.1 2.5-2.5s-1.1-2.5-2.5-2.5-2.5 1.1-2.5 2.5 1.1 2.5 2.5 2.5"></path></svg></div></div>

## Area issue was found in:
Workspace

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Ensure that the interactive element has a semantic button role and a proper accessible name. Here, replace the role="presentation" to role="button" to announce the actionable role of a button. Alternatively, implement the control using native elements such as a button.
https://webaim.org/techniques/forms/controls#buttons

## Environment(s)
Windows Windows 11 - Firefox

## Screenshots/Videos
<img width="1861" height="962" alt="Image" src="https://github.com/user-attachments/assets/5a4a41c2-14bb-479b-891e-d150011cef8c" />

https://github.com/user-attachments/assets/a3a58bae-9e1a-4e43-a98f-39424c4c994c

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6996754
**Issue reported by:** NA
**Slack conversation:** NA
